### DISCUSSION TITLE
vuepress-plugin-blog2 example / guide

### DISCUSSION THREAD

**Discussion by vmihailenco** [vuepress-theme-hope/vuepress-theme-hope#1628] [Q&A] [answered]
Are there any examples how to use vuepress-plugin-blog2 with basic theme?

The [guide](https://vuepress-theme-hope.github.io/v2/blog/guide.html#collecting-articles-and-generating-info) completely skips basic info like that and instead explains some advanced topics...

**Comment by Mister-Hope** [comment, marked as answer]
The plugin is facing theme developers, like `@vuepress/plugin-git` `@vuepress/plugin-shiki` and so on.

You should under stand vuepress structure and plugin api before using it. If so, the guide should be enough as the 2 example above is exaplaining to this level either.

If you are not expert at front-end area, you'd better search a theme which supports it and use directly.

**Comment by Mister-Hope** [comment]
BTW, I do have plans adding a demo showing how to use `vuepress-plugin-blog2` on default theme. Please wait for that patiently as this is an open source proejct and I do not gain income from it.
